applications/asset_tracker_v2/src/modules/modem_module.c

```diff
@@ -244,6 +244,28 @@ static void lte_evt_handler(const struct lte_lc_evt *const evt)
 	case LTE_LC_EVT_LTE_MODE_UPDATE:
 		nw_mode_latest = evt->lte_mode;
 		break;
+	case LTE_LC_EVT_MODEM_EVENT:
+		LOG_DBG("Modem domain event, type: %s",
+			evt->modem_evt == LTE_LC_MODEM_EVT_LIGHT_SEARCH_DONE ?
+				"Light search done" :
+			evt->modem_evt == LTE_LC_MODEM_EVT_SEARCH_DONE ?
+				"Search done" :
+			evt->modem_evt == LTE_LC_MODEM_EVT_RESET_LOOP ?
+				"Reset loop" :
+			evt->modem_evt == LTE_LC_MODEM_EVT_BATTERY_LOW ?
+				"Low battery" :
+			evt->modem_evt == LTE_LC_MODEM_EVT_OVERHEATED ?
+				"Modem is overheated" :
+				"Unknown");
+
+		if (evt->modem_evt == LTE_LC_MODEM_EVT_RESET_LOOP) {
+			LOG_WRN("The Modem has detected a reset loop. LTE network attach is now "
+				"restricted for the next 30 minutes. Reboot the device or call the "
+				"XFACTORYRESET AT command to reset the modem to factory settings. "
+				"For more information see the "
+				"nRF91 AT Commands - Command Reference Guide.");
```

Comment:
**suggestion:** add link the the relevant section of the guide.

Reply:
Wasn't able to directly link to the specific section so I added a reference to the version and chapter instead.